storage_v2/v1adapter/translator_test.go

```diff
@@ -253,3 +253,46 @@ func TestV1TracesFromSeq2(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateBatchesFromModelTrace(t *testing.T) {
+	testCases := []struct {
+		name            string
+		jtrace          model.Trace
+		expectedbatches []*model.Batch
+	}{
+		{
+			name: "one trace one batch",
+			jtrace: model.Trace{
+				Spans: []*model.Span{
```

Comment:
I suggest having two spans here with different process/service-name. Then the output of V1TraceToOtelTrace (which you should be testing, rather than the internal function) should be ptrace.Traces with two separate Resources.

Reply:
alright